### DISCUSSION TITLE
COG icon not changing to blue color when update available

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#1321] [Bug Reports]
Was Running rAudio 20230523 (since updated to 20230525)

COG icon is colored white instead of blue when update is available.

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by kan-ibal** [comment]
What message do you get after attempt to push Update button? Should it is Offline you can edit
`/srv/http/bash/common.sh`
and swap in line 182
`ipOnline 8.8.8.8` with `ipOnline www.github.com`
and in line 193
`ping -c 1 -w 1 $1 &> /dev/null && return 0`  with `curl -s --head $1 &> /dev/null && return 0`

Some of internet providers block `ping` or access to external DNS services.

**Reply by Sagnasty** [reply]
Running rAudio 20230525
@kan-ibal - The update date button / function works fine.
It is the "update available" indication that is currently not working (COG colored blue).

**Comment by Sagnasty** [comment]
Problem still exists with rAudio 20230603

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

Jim

**Comment by Sagnasty** [comment]
Problem still exists with rAudio 20230609

Additional info: if you open the Addons page and then close the page, the COG icon will change from white to blue color (assuming an update is available).

Jim

**Comment by Sagnasty** [comment]
Problem still exists in rAudio 20230615

**Comment by Sagnasty** [comment]
Problem still exists in rAudio 20230620
